lib/tasks/server/livereload-server.js

```diff
@@ -81,6 +85,15 @@ module.exports = Task.extend({
 
   didChange: function(results) {
     var filePath = path.relative(this.project.root, results.filePath || '');
+    var appStyleResource = this.project.pkg.name + '.css';
+
+    var reloadPaths = function() {
```

Comment:
i believe the function can be removed, and rather reloadPaths calculated here directly.